CHANGELOG.develop

```diff
@@ -2605,6 +2605,7 @@ files (thanks to Daniel Nicolai)
 - Added package =helm-lsp=
   - ~SPC m g s~ to find symbol in current project
   - ~SPC m g S~ to find symbol in all projects
+  - ~SPC m t g e~ to show error list
```

Comment:
I think this statement is wrong, the new binding is `SPC p E` for `helm-diagnostics`.

Reply:
I will fix it.